lib/fetching/bot.js

```diff
@@ -102,7 +102,7 @@ Bot.prototype._reportListener = function(reportData) {
       if (reportData._media === 'twitter') {
         reportData.metadata.retweet ? reportData.tags.push('RT') : reportData.tags.push('NO_RT');
       }
-      if (reportData._media === 'crowdtangle') {
+      if (reportData._media === 'facebook') {
```

Comment:
Is this a user-facing string or just internal? I personally think it's more accurate to use "crowdtangle" for internal things because that's what it truly is. We just want to use "facebook" when showing a string to the trackers, because they don't actually know or care what crowdtangle is.

In any case, this doesn't seem to fix the particular string here; I still see "__CROWDTANGLE" unless maybe I need to reconfigure something in settings:

<img width="614" alt="Screen Shot 2020-09-08 at 11 12 21" src="https://user-images.githubusercontent.com/2047062/93278056-6de3c000-f778-11ea-9e1f-b4a114058671.png">

Reply:
Did you try doing `node install.js && gulp angular`? I think thats needed when front end changes are made.

Reply:
Ah, I did something similar `npm install` (which triggers `gulp build`) and that resolved it. Seems like `npx gulp angular` is faster, I didn't know about that one. None of this should be necessary in development but unfortunately it is for now 🙃